examples/llamaindex/Evaluate_RAG_with_LlamaIndex.ipynb

```diff
@@ -0,0 +1,550 @@
+{
```

Comment:
<p>Clear outputs for <code>pip install</code> and for <code>llamaindex</code> specifically can you pin the version? It's &lt;1.0 so they can and have made breaking API changes, so it would help keep the guide working.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>